cpp/daal/src/externals/core_threading_win_dll.cpp

```diff
@@ -469,6 +555,17 @@ DAAL_EXPORT int _daal_threader_get_max_threads()
     return _daal_threader_get_max_threads_ptr();
 }
 
+DAAL_EXPORT int _daal_threader_get_current_thread_index()
```

Comment:
we use a mix of int, int32_t etc in the file

Reply:
It depends on particular cases when we need to wrap some TBB functionality